Bolts/src/main/java/bolts/Task.java

```diff
@@ -924,4 +939,9 @@ private void runContinuations() {
     /* package */ TaskCompletionSource() {
     }
   }
+
+  private static Task<?> NULL = new Task<>(null);
+  private static Task<Boolean> TRUE = new Task<>((Boolean) true);
+  private static Task<Boolean> FALSE = new Task<>((Boolean) false);
```

Comment:
Not a big fan of these, since in different languages they translate to different things, maybe rename to `TASK_TRUE`, etc?